docs/src/occurrences.md

```diff
@@ -48,16 +48,6 @@ next!
 complete!
 ```
 
-## Exporting results
```

Comment:
I think it makes a lot of sense to drop the DataFrame conversions. But maybe it'd still be relevant to define a DataStream object, that would iterate over NamedTuples with the data? that would allow any user to materialize it into a table object, and give immediate Query integration.

Reply:
I will add this at some point, because it also gives integration in JuliaDB.